src/pages/userManagement/UserManagement.jsx

```diff
@@ -165,15 +165,15 @@ export default function UserManagement() {
             </div>
             {Boolean(postUserError) && (
               <CustomAlert
-                onClose={() => setPostUserError(null)}
+                onClose={clearPostUserError}
```

Comment:
Does this need to be an anonymous function?
() => clearPostUserError()
Or am I still getting this wrong lol?

Reply:
This should be ok since clearPostUserError can be invoked correctly with the arguments that onClose will provide.

Reply:
@Atticus29 this is actually preferable. if we don't need to create an extra function it's better not to do so (though the performance implications are quite negligible either way)